test/core/integration/concrete/lockup-base/withdraw-hooks/withdrawHooks.t.sol

```diff
@@ -62,11 +63,7 @@ contract WithdrawHooks_Integration_Concrete_Test is Integration_Test {
         });
 
         // Make the withdrawal.
-        lockup.withdraw({
-            streamId: differentSenderRecipientStreamId,
-            to: address(recipientGood),
-            amount: withdrawAmount
-        });
+        withdraw({ streamId: differentSenderRecipientStreamId, to: address(recipientGood), amount: withdrawAmount });
```

Comment:
The advantage of using `lockup.withdraw` with explicit `value` is that we can differentiate which tests is with 0 value and which are not. In the current approach, its hard to differentiate between the two by looking at the code.

Reply:
All calls to withdraw have value greater than zero. So not sure what you are referring to

Reply:
I see. I thought it is only in `withdraw` tests and not all calls. In that case, can we please have explicitness instead of using these helper functions? May be have non zero fee in withdraw test but 0 feel at all other places in Concrete.